examples/pytorch/image-classification/run_image_classification.py

```diff
@@ -325,21 +325,25 @@ def compute_metrics(p):
         size = image_processor.size["shortest_edge"]
     else:
         size = (image_processor.size["height"], image_processor.size["width"])
-    normalize = Normalize(mean=image_processor.image_mean, std=image_processor.image_std)
+    normalize = (
+        [Normalize(mean=image_processor.image_mean, std=image_processor.image_std)]
+        if hasattr(image_processor, "image_mean")
+        else []
+    )
```

Comment:
Let's just define an identity function rather than having to create a list which we then expand

```suggestion
    if hasattr(image_processor, "image_mean"):
        normalize = Normalize(mean=image_processor.image_mean, std=image_processor.image_std)
    else:
        normalize = transforms.Lambda(lambda x: x)
```

Reply:
@amyeroberts maybe better `normalize = Normalize(mean=image_processor.image_mean, std=image_processor.image_std) if hasattr(image_processor, "image_mean") else transforms.Lambda(lambda x: x)` in case someone would print sourcecode, one or two lines would be saved ;-)